### PR TITLE
Library updates

### PR DESCRIPTION
**Pull request by adkumar1**
- Spring boot library upgrade
 - Spring cloud library upgrade

Note: Please merge PR #7  before this

### CODE REVIEW

File: DEPENDENCIES [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-maven/mavencentral/jakarta.activation/jakarta.activation-api/2.1.1, EPL-2.0 OR BSD-3-Clause OR GPL-2.0-only with Classpath-exception-2.0, approved, ee4j.jaf
+maven/mavencentral/jakarta.activation/jakarta.activation-api/2.1.2, EPL-2.0 OR BSD-3-Clause OR GPL-2.0-only with Classpath-exception-2.0, approved, ee4j.jaf
 maven/mavencentral/jakarta.annotation/jakarta.annotation-api/2.1.1, EPL-2.0 OR GPL-2.0-only with Classpath-exception-2.0, approved, ee4j.ca
 maven/mavencentral/jakarta.validation/jakarta.validation-api/3.0.2, Apache-2.0, approved, clearlydefined
 maven/mavencentral/jakarta.xml.bind/jakarta.xml.bind-api/4.0.0, BSD-3-Clause, approved, ee4j.jaxb
 maven/mavencentral/org.apache.commons/commons-lang3/3.12.0, Apache-2.0, approved, clearlydefined
-maven/mavencentral/org.apache.logging.log4j/log4j-api/2.19.0, Apache-2.0, approved, clearlydefined
-maven/mavencentral/org.apache.logging.log4j/log4j-to-slf4j/2.19.0, Apache-2.0, approved, #5941
+maven/mavencentral/org.apache.logging.log4j/log4j-api/2.20.0, Apache-2.0, approved, clearlydefined
+maven/mavencentral/org.apache.logging.log4j/log4j-to-slf4j/2.20.0, Apache-2.0, restricted, clearlydefined
```
Comment (Siegfriedk) [inline DEPENDENCIES]: You have restriced here which means (as far as i understand it) that you need to have a committer creating ip issues for you. You can also wait until someone else did it and the dash tool is able to provide an ticket.

i don't have time this week to do that, pls ask your committer to do so

**Review by Siegfriedk — CHANGES_REQUESTED** [review]

File: DEPENDENCIES (hunk 1 above)
Comment (adkumar1) [reply DEPENDENCIES]: @fedornaz

Comment (Wulghash) [reply DEPENDENCIES]: @SebastianBezold @wjost Can you please do it?

Comment (Wulghash) [reply DEPENDENCIES]: @Siegfriedk It got resolved, please check.

**Pull request merged**